src/mixxx.cpp

```diff
@@ -135,6 +137,13 @@ inline QLocale inputLocale() {
             QLocale(QLocale::English);
 }
 
+#ifdef __LINUX__
+inline bool supportsGlobalMenu() {
+    QDBusConnection conn = QDBusConnection::sessionBus();
+    return conn.interface()->isServiceRegistered("com.canonical.AppMenu.Registrar");
```

Comment:
The Qt Implementation has also a null check. 
https://github.com/qt/qtbase/blob/6e2963200642fa0171b247cb986726ca124795b0/src/gui/platform/unix/qgenericunixthemes.cpp#L108

Reply:
For completeness you may also check the QT_NO_DBUS macro.